src/views/virtualmachines/details/tabs/configuration/network/components/modal/VirtualMachinesNetworkInterfaceModal.tsx

```diff
@@ -50,23 +45,16 @@ const VirtualMachinesNetworkInterfaceModal: FC<VirtualMachinesNetworkInterfaceMo
 
         const updatedNetworks: V1Network[] = [...(getNetworks(vm) || []), resultNetwork];
         const updatedInterfaces: V1Interface[] = [...(getInterfaces(vm) || []), resultInterface];
-        const updatedVM = updateVMNetworkInterface(vm, updatedNetworks, updatedInterfaces);
+        const updatedVM = updateVMNetworkInterfaces(vm, updatedNetworks, updatedInterfaces);
 
-        if (nicHotPlugEnabled && interfaceType === interfacesTypes.bridge) {
-          return addInterface(updatedVM, {
```

Comment:
Do you think we can even remove the methods `addInterface` and `removeInterface` ?

Reply:
Yes! Thanks for the reminder. Done. =)